[2021-06-16 13:06:47,548] {taskinstance.py:876} INFO - Dependencies all met for <TaskInstance: summary_db.ambassador_refunds 2021-06-15T00:00:00+00:00 [queued]>
[2021-06-16 13:06:47,584] {taskinstance.py:876} INFO - Dependencies all met for <TaskInstance: summary_db.ambassador_refunds 2021-06-15T00:00:00+00:00 [queued]>
[2021-06-16 13:06:47,584] {taskinstance.py:1067} INFO - 
--------------------------------------------------------------------------------
[2021-06-16 13:06:47,584] {taskinstance.py:1068} INFO - Starting attempt 2 of 5
[2021-06-16 13:06:47,584] {taskinstance.py:1069} INFO - 
--------------------------------------------------------------------------------
[2021-06-16 13:06:47,798] {taskinstance.py:1087} INFO - Executing <Task(PythonOperator): ambassador_refunds> on 2021-06-15T00:00:00+00:00
[2021-06-16 13:06:47,811] {standard_task_runner.py:52} INFO - Started process 1434 to run task
[2021-06-16 13:06:47,825] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'summary_db', 'ambassador_refunds', '2021-06-15T00:00:00+00:00', '--job-id', '212', '--pool', 'default_pool', '--raw', '--subdir', 'DAGS_FOLDER/first_dag.py', '--cfg-path', '/tmp/tmpftw35wg8', '--error-file', '/tmp/tmp_6qehtbj']
[2021-06-16 13:06:47,825] {standard_task_runner.py:77} INFO - Job 212: Subtask ambassador_refunds
[2021-06-16 13:06:47,984] {logging_mixin.py:104} INFO - Running <TaskInstance: summary_db.ambassador_refunds 2021-06-15T00:00:00+00:00 [running]> on host ac3e7c1eb084
[2021-06-16 13:06:48,129] {taskinstance.py:1282} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=summary_db
AIRFLOW_CTX_TASK_ID=ambassador_refunds
AIRFLOW_CTX_EXECUTION_DATE=2021-06-15T00:00:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2021-06-15T00:00:00+00:00
[2021-06-16 13:06:48,131] {logging_mixin.py:104} INFO - ------------------------------------Inside Function-----------------------------
[2021-06-16 13:07:17,600] {logging_mixin.py:104} INFO - -------------------------------Data read into dataframe---------------------------
[2021-06-16 13:07:19,682] {logging_mixin.py:104} INFO - ----------------------------------Data written into summary DB----------------------
[2021-06-16 13:07:19,682] {python.py:151} INFO - Done. Returned value was: None
[2021-06-16 13:07:19,703] {taskinstance.py:1191} INFO - Marking task as SUCCESS. dag_id=summary_db, task_id=ambassador_refunds, execution_date=20210615T000000, start_date=20210616T130647, end_date=20210616T130719
[2021-06-16 13:07:19,765] {taskinstance.py:1245} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2021-06-16 13:07:19,776] {local_task_job.py:151} INFO - Task exited with return code 0
